src/main/java/game/race/dto/RaceDto.java

```diff
@@ -0,0 +1,50 @@
+package game.race.dto;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RaceDto {
+
+    private int tryCnt;
+    private int vehicleCnt;
+    private List<String> vehicleNames;
+
```

Comment:
```suggestion

    private int tryCnt;
    private List<String> vehicleNames;
```

vehicleNames에서 이름 리스트의 사이즈를 알 수 있으니 차량 수는 굳이 안 담아도 될 것 같습니다.